ChatApp/Custom+UI/SenderChatBubbleView.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  SenderChatBubbleView.swift
+//  ChatApp
+//
+//  Created by bakhva  on 27.04.23.
+//
+
+import UIKit
+
+class SenderChatBubbleView: UIView {
+    
+    //MARK: Components
+    lazy var color: UIColor = ChatAppColors.senderBubbleColor
+    
+    private lazy var mainTextBubble: UIView = {
+        let bubble = UIView()
+        bubble.translatesAutoresizingMaskIntoConstraints = false
+        bubble.backgroundColor = color
+        bubble.layer.cornerRadius = SenderChatBubbleViewConstants.mainTextBubbleCornerRadius
+        return bubble
+    }()
+    
+    lazy var messageLabel: UILabel = {
```

Comment:
```suggestion
    let messageLabel: UILabel = {
```

also consider making them private.
e.g. if we only consider that outside class can only change text - it should not be allowed to change font color or something

P.S. this is for every subview which is not private